### ISSUE TITLE
"Tag, You're It!" achievement does not work

### ISSUE THREAD

**Issue by MADAFACA078** [Cigaras/Payday-2-Solo-Achievements#1]
This achievement "Tag, You're It!"(In the Car Shop job, be the first player in a four-man crew
to reach the docks.) is not working for me with or without AI. I tried to order AIs stay in car 
shop then drive to dock by myself. It still not work. I used this mod to achieve many 
achievement that need more than one players. But this one is not working to me. 
Thank you for creating such a great mod, I hope you can help me fix this problem. 
In the end, thank you again and have a good day!

**Comment by Cigaras** [comment]
Sorry, I can not find anything in available lua files related to this achievement exept it's name - fort_3 - so I can not mod it 🙁

I myself did it with random people because coordination is not necessary: I just started a public game alone, over the course of heist random people popped-in, once C4 was set up I had a full party, so I grabbed the keys, rushed to the car and drove off. In the meantime one player died and got into custody, another decided to head tot he docks by foot, so I was left to race with one player only. We reached the docks, I got the achievement, kicked the ones left behind and completed the heist. I was lucky to get it on first try, to be honest I wasn;t even going for the achievement, so it might take You more than one run, but at least it does not require specific actions from players like Short Fuse, Pass the Ammo and similar achievements.
